docs/audits.md

```diff
@@ -372,7 +372,7 @@ in their `uses:` are subject to a degree of ambiguity: a ref like
 
 An attacker can exploit this ambiguity to publish a branch or tag ref that
 takes precedence over a legitimate one, delivering a malicious action to
-pre-existing consumers of that action without having to modify those consumers.
```

Comment:
Could you undo these ones? I prefer pre-existing over preexisting (and the AP Stylebook agrees that it's an idiomatic spelling).

(CMOS and Fowler's Modern English disagree but weakly.)